doc/teacher.md

```diff
@@ -155,3 +158,44 @@ Comments can be left to every criterion checkpoint or to the whole group of grad
 **Release** page gives access to the overview of the results of manual reviews for all course students and assignments. There the user can see submission score for each review, the mean score for all ready reviews, set a final grade/status for a student submission (**Approved/Rejected/Revision**), look at all available reviews for each submission, and *release* the results to reveal them to students or student groups.
 
 It is also possible to mass approve submissions or mass release reviews for an assignment by choosing a minimal score and then pressing `Approve all` or `Release all` correspondingly. Every submission with a score equal or above the set minimal score will be approved or reviews to such submissions will be released.
+
+Grading criteria can be loaded from a file `criteria.json` in a corresponding assignment folder inside the `Tests` repository. 
+
+JSON format: 
+
+```
+[
+    {
+        "heading": "First criteria group",
+        "criteria": [
+            {
+                "description": "Has headers",
+                "score": 5
+            },
+            {
+                "description": "Has footers",
+                "score": 10
+            }
+        ]
+    },
+    {
+        "heading": "Second criteria group",
+        "criteria": [
+            {
+                "description": "Has forms",
+                "score": 5
+            },
+            {
+                "description": "Has inputs",
+                "score": 5
+            },
+            {
+                "description": "Looks nice",
+                "score": 10
+            }
+        ]
+    }
+]
+```
+
+`score` field is optional. If set, the max score for the assignment will be equal to the sum of all scores for each criteria. Otherwise, the max total score will be 100%.
```

Comment:
Missing newline.

To avoid confusion, I wonder if this should be `max_score` instead, since this is a declaration of scoring criteria.
If not set, I think this should mean that `each criterion counts equally towards the total score`, or something like that.

Reply:
But it is the score for a single criterion, not the max score for the whole set of criteria? Isn't the name `max_score` equally confusing_

Reply:
Updated the docs.

Reply:
First of all, I don't see any changes in the docs as mentioned above. Did something happen to the commit?

Second, the problem I have with this is that these criterion definitions use the same terminology as in the [`kit/score.Score`](https://github.com/autograde/quickfeed/blob/725ddfa95cf3ac20c53d7d698473dc31a72dd158/kit/score/score.go#L33) type, where `MaxScore` is defined for one score object, and the `Score` can take any value in the range `[0,MaxScore]`.

The question I have is what these `score` lines in the JSON format really mean?

Do they each define the maximum score that can be awarded for one of these criterion?

Or looking at this now, it appears as if these are binary answers, each giving `score` points. Meaning that each criterion either gives `0` or `score` points, and nothing in-between. If that's the case, then maybe use the term `points` to distinguish this from the other use of `score`. 

Mapping this to the `Score` type:

```go
sumPoints := <..>
sc := score.NewMaxScore(testName, sumPoints, weight)
// for each incorrect criterion
sc.DecBy(criterionPoints)
```

(currently the signature shown below won't work, but in a future update it will)

PS: Don't know if these JSON things are even mapped to score objects, but either way it makes sense to avoid confusing the terminology.

PPS: Note that `criterion` is singular, and `criteria` is plural.

Reply:
No, they are not mapped to score objects. Agree that `points` is a better option. Renamed.